File: app/src/main/java/com/akexorcist/localizationapp/viewpager/HelloFragment.java
package com.akexorcist.localizationapp.viewpager;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.akexorcist.localizationapp.R;

/**
 * Created by Akexorcist on 7/22/15 AD.
 */
public class HelloFragment extends Fragment {

    public static Fragment newInstance() {
        return new HelloFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_hello, container, false);
    }
}

File: app/src/main/java/com/akexorcist/localizationapp/viewpager/HelloFragment.kt
package com.akexorcist.localizationapp.viewpager

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.akexorcist.localizationapp.databinding.FragmentViewPagerHelloBinding

class HelloFragment : Fragment() {
    companion object {
        @JvmStatic
        fun newInstance(): Fragment = HelloFragment()
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        val binding = FragmentViewPagerHelloBinding.inflate(layoutInflater, container, false)
        return binding.root
    }
}
